docs/tutorials/toy-connector.md

```diff
@@ -265,23 +267,18 @@ def _call_api(endpoint, token):
 
 
 def check(config):
-    # Assert required configuration was provided
```

Comment:
this assertion doesn't do anything. the spec says that this field is required so this branch will never trigger. prefer to remove since we want less content for the tutorial where possible.